Mathlib/Algebra/Module/Submodule/Range.lean

```diff
@@ -350,6 +352,16 @@ theorem map_subtype_embedding_eq (p' : Submodule R p) :
   rfl
 #align submodule.map_subtype_embedding_eq Submodule.map_subtype_embedding_eq
 
+/-- If `N ⊆ M` then submodules of `N` are the same as submodules of `M` contained in `N`. -/
+def mapIic [Semiring R] [AddCommMonoid M] [Module R M] (p : Submodule R M) :
```

Comment:
I don't understand this name. (But I also don't understand the name `MapSubtype` above.)

Reply:
The idea of the name is that we are mapping submodules of a submodule `p` into the interval `Iic`, of submodules bounded above by `p`.